packages/react-composites/src/composites/CallComposite/utils/Utils.test.ts

```diff
@@ -35,3 +35,39 @@ describe('SDKUtils tests', () => {
     });
   });
 });
+
+describe('Record and Transcription tests', () => {
+  describe('computeVariant tests', () => {
+    test('computeVariant should return expected values', () => {
```

Comment:
can we have more granular unit tests. Don't just bunch up everything into one big test.

Reply:
instead of having comments for each "test", we should just use separate test blocks where we can just make the comment into the string of what the test is doing.

Reply:
@carocao-msft For example
`computeVariant should start recording`
`computeVariant should stop recording`
et etc. :)

Reply:
Updated unit test :)